packages/dds/tree/src/simple-tree/core/walkSchema.ts

```diff
@@ -31,22 +41,23 @@ export function walkNodeSchema(
  * Traverses all {@link TreeNodeSchema} schema reachable from `allowedTypes`, applying the visitor pattern.
  */
 export function walkAllowedTypes(
-	allowedTypes: Iterable<TreeNodeSchema>,
+	annotatedAllowedTypes: NormalizedAnnotatedAllowedTypes,
 	visitor: SchemaVisitor,
 	visitedSet: Set<TreeNodeSchema> = new Set(),
 ): void {
-	for (const childType of allowedTypes) {
-		walkNodeSchema(childType, visitor, visitedSet);
+	for (const annotatedAllowedType of annotatedAllowedTypes.types) {
+		const { type } = annotatedAllowedType;
+		walkNodeSchema(type, visitor, visitedSet);
 	}
-	visitor.allowedTypes?.(allowedTypes);
+	visitor.allowedTypes?.(annotatedAllowedTypes);
 }
 
 /**
  * Callbacks for use in {@link walkFieldSchema} / {@link walkAllowedTypes} / {@link walkNodeSchema}.
  */
 export interface SchemaVisitor {
 	/**
-	 * Called once for each node schema.
+	 * Called for each node schema.
```

Comment:
Is there a reason to remove "once" from the docs here? This isn't intended to imply a behavior change is it?

Reply:
Ah, this was just because I had changed it and then changed it back.